cmd/containerd/trace.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"github.com/sirupsen/logrus"
+	"go.opencensus.io/trace"
+)
+
+type logrusExporter struct{}
+
+func (le *logrusExporter) ExportSpan(s *trace.SpanData) {
+	logrus.WithFields(logrus.Fields{
+		"name":          s.Name,
+		"startTime":     s.StartTime,
+		"endTime":       s.EndTime,
+		"duration":      s.EndTime.Sub(s.StartTime),
+		"traceID":       s.TraceID,
+		"spanID":        s.SpanID,
+		"parentSpanID":  s.ParentSpanID,
+		"statusCode":    s.Code,
+		"statusMessage": s.Message,
+		"attributes":    s.Attributes,
```

Comment:
Does `logrus` support `slice` by default? Should `fmt.Sprintf` these?

Reply:
I think slice support varies by what formatter/hook is used. For reference this is what I see with TextFormatter:

```
time="2019-05-24T11:01:40.340518400-07:00" level=debug msg=Span attributes="map[A:first B:second]" duration=0s endTime="2019-05-24 11:01:40.3405184 -0700 PDT m=+0.020996201" name=Foo parentSpanID=0000000000000000 spanID=bde151dbed46d67a startTime="2019-05-24 11:01:40.3405184 -0700 PDT m=+0.020996201" statusCode=0 statusMessage= traceID=4e553c73a8c28296b31f2e9843a40c9f
```

And this with JSONFormatter:

```
{"attributes":{"A":"first","B":"second"},"duration":0,"endTime":"2019-05-24T11:05:22.5578808-07:00","level":"debug","msg":"Span","name":"Foo","parentSpanID":[0,0,0,0,0,0,0,0],"spanID":[93,31,81,236,252,38,239,78],"startTime":"2019-05-24T11:05:22.5578808-07:00","statusCode":0,"statusMessage":"","time":"2019-05-24T11:05:22-07:00","traceID":[30,196,213,131,209,97,137,212,210,131,134,42,109,102,214,230]}
```

This was the sample span code:

```
		_, span := trace.StartSpan(gocontext.Background(), "Foo")
		span.AddAttributes(trace.StringAttribute("A", "first"), trace.StringAttribute("B", "second"))
		defer span.End()
```

Reply:
Just noticed it's kind of gross that the span and trace ID print that way for JSONFormatter. Think I should log them as strings. Thoughts?

Reply:
Yea strings for sure! But it looks like they have already implemented `Stringer` so we should just call `s.TraceID.String()`, `s.SpanID.String()`

Reply:
Good catch :) Switched to using `.String()`